examples/SpringTxFailoverExample/src/main/java/example/ApiController.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import software.amazon.jdbc.plugin.failover.FailoverSQLException;
+
+@RestController
+@EnableRetry
```

Comment:
Im not too familiar with Spring Retry, but a lot of the examples/tuts i see put the `@EnableRetry` in a configuration class with the `@Configuration` annotation. 
Also, most examples have the `@Retryable` annotation directly above specific methods. 
Does having these annotations here in the controller class work?

Reply:
It still works with annotations in the controller class, but I've moved them to individual methods/configuration class.